CMakeLists.txt

```diff
@@ -1208,6 +1244,32 @@ set(CPACK_WIX_PROPERTY_ARPHELPLINK "${CPACK_PACKAGE_HOMEPAGE_URL}")
 set(CPACK_WIX_UI_BANNER "${CMAKE_CURRENT_SOURCE_DIR}/build/wix/images/banner.bmp")
 set(CPACK_WIX_UI_DIALOG "${CMAKE_CURRENT_SOURCE_DIR}/build/wix/images/dialog.bmp")
 
+set(CPACK_BUNDLE_NAME "Mixxx")
+set(CPACK_BUNDLE_ICON "${CMAKE_CURRENT_SOURCE_DIR}/res/osx/VolumeIcon.icns")
+set(CPACK_BUNDLE_APPLE_ENTITLEMENTS "${CMAKE_CURRENT_SOURCE_DIR}/build/osx/entitlements.plist")
+set(MACOS_BUNDLE_VERSION "${CMAKE_PROJECT_VERSION}")
+set(MACOS_BUNDLE_SHORTVERSION "${CMAKE_PROJECT_VERSION}")
+set(MACOS_BUNDLE_MINOSVERSION "10.11.0")
```

Comment:
I thought the minimum macOS version was 10.13?

Reply:
Apparently not, I took that from https://github.com/mixxxdj/mixxx/blob/master/build/osx/product_definition.plist#L11, which is parsed here: https://github.com/mixxxdj/mixxx/blob/master/build/util.py#L194-L206

Reply:
Hmm, I'm not sure why I was thinking 10.13. https://mixxx.org/download/ says 10.11 as well. IIRC the Jenkins server runs 10.13?